lib/pages/search_tab.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+
+import '../hooks/stores.dart';
+import '../util/goto.dart';
+import '../widgets/bottom_modal.dart';
+import 'search_results.dart';
+
+class SearchTab extends HookWidget {
+  const SearchTab();
+
+  @override
+  Widget build(BuildContext context) {
+    final searchInputController = useTextEditingController();
+
+    final accStore = useAccountsStore();
+    final instance = useState(accStore.instances.first);
+    useValueListenable(searchInputController);
+
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.transparent,
+        shadowColor: Colors.transparent,
+      ),
+      body: GestureDetector(
+        onTapDown: (_) => primaryFocus.unfocus(),
+        child: ListView(
+          padding: const EdgeInsets.symmetric(horizontal: 20),
+          children: [
+            TextField(
+              controller: searchInputController,
+              autofocus: true,
+              textAlign: TextAlign.center,
+              decoration: InputDecoration(
+                fillColor: Colors.grey,
+                isDense: true,
+                // border: InputBorder.none,
+                contentPadding: const EdgeInsets.symmetric(vertical: 10),
+                hintText: 'search',
+                border: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(10),
+                ),
+              ),
+            ),
+            const SizedBox(height: 5),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              // crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                Expanded(
+                  child: Text('instance:',
+                      style: Theme.of(context).textTheme.subtitle1),
+                ),
+                Expanded(
+                  child: SelectInstanceButton(
+                    initial: instance.value,
+                    onChange: (s) => instance.value = s,
+                  ),
+                ),
+              ],
+            ),
+            if (searchInputController.text.isNotEmpty)
+              ElevatedButton(
+                onPressed: searchInputController.text.isEmpty
```

Comment:
`searchInputController.text.isNotEmpty` and `searchInputController.text.isEmpty` exclude each other